### ISSUE TITLE
Group users by teams

### ISSUE THREAD

**Issue by mathieumd** [phpList/phplist3#929]
It seems that currently, there are only two type of users (*Super Administrators* and *Administrators*). And without being SuperAdmin, there is no way to share lists and campaign, right?

We would like a way to let user A access lists created by user B, and vice-versa. In short, adding users to a group/team which would share all resources attached to it, without being SuperAdmin themselves.

**Comment by michield** [comment]
Yes, interesting. I guess with the current admin arrangement that's not easy. Maybe for now, each team has a shared login. Maybe we can make the "admin" in the system more like a team and have different logins associate with that team. That's not a trivial change though.

**Comment by mathieumd** [comment]
Yes, for now we will use a single shared login. I guess most organization do like that today (or maybe they deploy several instances, but that's overkill and a pain to maintain)

As I see it (but I have not digged into the code), it could also be done by adding a field "team" (or maybe better, "teams") to each objects (Lists, Campaigns, etc.) and admins (it would be useless for Super-admin, they would still see all and everything), and allow their access only to matching admins.

**Comment by hilburger** [comment]
Would love to see such a feature, too. <3
It's really crucial to software usage in organizations with multiple content owners or teams. 

Grouping in teams seems like a slim and easy to undrstand way of organizing editors.
